app/controllers/posts_controller.rb

```diff
@@ -3,13 +3,13 @@ def index
     @user = current_user
     @posts = Post.where(user: params[:user_id])
     @all_posts = Post.where(user: params[:user_id])
-    @comments = Comment.where(post_id: params[:user_id])
+    @comments = Comment.includes([:user]).where(post_id: params[:user_id])
```

Comment:
- [ ] As specified in the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md), " `Make sure that the N+1 problem is solved when fetching all posts and their comments for a user. ` ", since we are fetching all posts and their comments for a particular user and not all comments for a particular posts and it's owner,  I think a better a way to solve the `N+1` will be like so: 
```
posts = @user.posts.includes(:comments) 
```
Please kindly fix that.